src/natcap/invest/utils.py

```diff
@@ -482,9 +476,13 @@ def build_lookup_from_csv(
     if col_list and key_field not in col_list:
         col_list.append(key_field)
    
+   # utf-8-sig will work whether or not there is a BOM
     table = pandas.read_csv(
         table_path, sep=None, index_col=False, engine='python', 
-        encoding=encoding)
+        encoding='utf-8-sig')
```

Comment:
In reading about Unicode I learned that utf-8-sig works for decoding with or without a BOM, so it shouldn't be necessary to check for it here. (https://docs.python.org/3/howto/unicode.html#reading-and-writing-unicode-data)

Reply:
Great!  That whole BOM encoding thing was mighty frustrating for a long time, and if `utf-8-sig` handles the BOM if it's there, then that'll help simplify quite a few things!  I added an issue for this at https://github.com/natcap/invest/issues/264

Reply:
Update: I'm no longer sure about this. `utf-8-sig` seems to work for UTF-8 with or without a BOM, but not for any other encodings. So `test_csv_not_utf_8` was failing. Weirdly this test passes with `encoding=None`. But `encoding=None` does not work when there is a BOM. I spent some time reading about this and still not sure what's going on. I don't think pandas guesses what the encoding is, so maybe the 'iso8859_5' Cyrillic encoding used in the test just happens to be interpretable as UTF-8?

Reply:
Uffda, encodings (and localization more generally) are quite the rabbit hole!  Thanks for the background and for looking into this.  The behavior you're describing does sound a bit bizarre ... I'm sorry I don't have any magic solutions here, and guessing encodings can be problematic at best (I highly doubt `pandas` is doing any guessing of encodings).  It's quite possible that the strings in the cyrrilic test are valid UTF-8 (if gibberish), and if that's true then there wouldn't be an error when decoding as UTF-8.

Fortunately, most of the issues we've encountered are merely with the presence or absence of the BOM, so we might be able to defer the true solution to the encodings issue until later.  There are other localization issues at hand, too, (example: `196,96 00` --> `196.96`) so we'll want to take a look at how we can offer more reliable interpretation of tabular inputs some time down the road and probably in the workbench.

Side note: If you're ever looking for some background information on encodings, the first couple chapters of https://www.oreilly.com/library/view/fonts-encodings/9780596102425/ are _awesome_ background and history on the subject.